src/neuroglancer/segmentation_display_state/frontend.ts

```diff
@@ -128,6 +130,25 @@ export function getObjectColor(
   const color = tempColor;
   color[3] = alpha;
   displayState.segmentColorHash.compute(color, objectId);
+
```

Comment:
I think you can adjust saturation by just interpolating the color with (1, 1, 1), as is done for the isSelected case.